#!/usr/bin/env python
PACKAGE = "image_cloud"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("subscribe_topic_image1",		str_t,	 	0, "topic image1", "")
gen.add("subscribe_topic_image2",		str_t,	 	0, "topic image2", "")
gen.add("publish_topic_image1",			str_t,		0, "remapped image topic1", "" )
gen.add("publish_topic_image2",			str_t,		0, "remapped image topic2", "" )
gen.add("queue_size",					int_t,		0, "message filter queue", 30, 1, 100 )
gen.add("publish_image2",				bool_t,		0, "republish image 2", True)

exit(gen.generate(PACKAGE, "image_cloud", "sync_image"))